spotbugs-tests/src/test/java/edu/umd/cs/findbugs/TextUICommandLineTest.java

```diff
@@ -11,24 +11,34 @@
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 class TextUICommandLineTest {
     @Test
-    void handleOutputFileReturnsRemainingPart() throws IOException {
+    void duplicateFilepathIsIgnored() throws IOException {
```

Comment:
I would love to see one more test, where there are two different files are added with the same commandline reporter.